playbook/workshops/contrib/ajar/README.md

```diff
@@ -234,4 +234,174 @@ You've finished Part 1!
 
 ![](img/celebrate.gif)
 
-# More coming soon...
+
+## Part 2
+
+Welcome to part 2 of the Ajar.io workshop!
+
+To recap, in [part 1](#part-1) you learned about:
+
+- **JavaScript variables**: used to store data values. Declared using the `var`
+  keyword.
+- **Drawing on the canvas**: the HTML `<canvas>` element is used to draw
+  graphics, on the fly, via scripting (usually JavaScript). You obtain the 
+  drawing context of the `<canvas>` element by calling the _function_ (more on
+  what that means today) `getContext("2d")` of the `<canvas>` element. You use
+  the drawing context to draw shapes on the canvas.
+
+In this part you will learn about **functions**.
+
+### What are functions?
+
+In part 1, you wrote all your code outside of any functions. This means that
+your code runs from *top to bottom*. This was okay for a simple task such as
+displaying one circle on screen. For more complex scenarios we will need 
+**functions**.
+
+**A JavaScript function is a block of code designed to perform a particular
+task.**
+
+A function looks like this:
+
+```
+function name() {
+    // code to be executed goes here <-- that is a comment
+}
+```
+
+A JavaScript function is defined with the `function` keyword, followed by a name,
+followed by parentheses `()`. The code to be executed by the function is
+placed inside curly brackets: `{}`.
+
+You "run" a function (the code inside of it) by "invoking" the function.
+
+You use the `()` operator invokes the function. To invoke the function `name`
+from above, you would write the code:
+
+```
+name();
+```
+
+The code contained within the curly brackets `{}` would then be executed.
+
+Unlike the code in part 1, **code contained in functions can be run multiple
+times**.
+
+Let's make another function.
+
+```
+function addOne(myNumber) {
+    myNumber = myNumber + 1;
+}
+```
+
+This function's name is `addOne`.
+
+But look! There's something inside of the parentheses. You can put
+**parameters** inside function parentheses. Parameters are values you can give 
+to the function that the function can then use when executing it's code block.
+
+In this function we give the function a number. The function can refer to this
+number as `myNumber`.
+
+The function proceeds to take the number (`myNumber`) and add one to it.
+
+So how do we "invoke" this function? Like this:
+
+```
+addOne(5);
+```
+
+The function thinks of `myNumber` as a variable whose value is `5`. It then
+proceeds to run the code:
+
+```
+myNumber = myNumber +1;
+```
+
+Because `myNumber` is 5, you can think of the computer seeing the code like
+this:
+
+```
+myNumber = 5 + 1;
+```
+
+So `myNumber` becomes `6`.
+
+### Using functions in Ajar.
+
+Right now your code should look like this:
+> `Index.html`
+> ```
```

Comment:
lol. I zoned.

What I meant was to set up syntax highlighting (left a note on the PR about it).